README.md

```diff
@@ -66,13 +66,14 @@ npm start
 To run the backend (port 4000) alongside the frontend, also use:
 
 ```
-//first comment out lines 12 & 13 in /src/index.jsx
-
+// Run the backend server
 cd backend
-
 npm i
-
 npm start
+
+// Then run the frontend with the following:
+cd ..
```

Comment:
I guess this doesn't make a lot of sense since the dev has to open a new terminal after starting the backend on one terminal